.github/BOTMETA.yml

```diff
@@ -851,30 +1015,31 @@ files:
   test/integration/targets/aci:
     maintainers: $team_aci
     labels:
-    - networking
+      - networking
   test/integration/targets/meraki:
     maintainers: $team_meraki
     labels:
-    - networking
+      - networking
   test/integration/targets/nxos:
     maintainers: $team_nxos
     labels:
-    - networking
+      - networking
   test/integration/targets/setup_acme:
     maintainers: resmo felixfontein
   test/integration/targets/setup_zabbix:
     maintainers: eikef D3DeFi
   test/integration/targets/ucs:
     maintainers: $team_ucs
     labels:
-    - remote_management
+      - remote_management
   test/units/modules/network:
     maintainers: $team_networking
     labels: networking
   test/units/modules/network/nxos:
     maintainers: $team_nxos
     labels:
-    - networking
+      - nxos
```

Comment:
No need to add `nxos` since it's a path component. When the path is `test/units/modules/network/nxos`, the bot will automatically adds the following labels (if they exists): `test`, `units`, `modules`, `network`, `nxos`.

Reply:
Which brings us neatly to the following:
Should we add a hierarchy to integration tests, so they can be auto-labeled too ?

Reply:
Using Ansibullbot to add labels for tests (targets, legacy and units) is tracked in https://github.com/ansible/ansibullbot/issues/808#issuecomment-419853973